app/assets/javascripts/discourse/app/lib/uppy-checksum-plugin.js

```diff
@@ -0,0 +1,93 @@
+import { Plugin } from "@uppy/core";
+import { Promise } from "rsvp";
+
+export default class UppyChecksum extends Plugin {
+  constructor(uppy, opts) {
+    super(uppy, opts);
+    opts = opts || {};
```

Comment:
```suggestion
    opts = opts || {};
    super(uppy, opts);
```

Should the default value be assigned before the `super` function call?

Reply:
Good call, don't even need to do this because the Plugin class does it for you